contrib/IECoreAppleseed/src/IECoreAppleseed/ShaderNetworkAlgo.cpp

```diff
@@ -65,13 +65,9 @@ namespace ShaderNetworkAlgo
 
 renderer::ShaderGroup *convert( const IECoreScene::ShaderNetwork *shaderNetwork )
 {
-	ShaderNetworkPtr networkCopy;
-	if( true ) // todo : make conditional on OSL < 1.10
-	{
-		networkCopy = shaderNetwork->copy();
-		IECoreScene::ShaderNetworkAlgo::convertOSLComponentConnections( networkCopy.get() );
-		shaderNetwork = networkCopy.get();
-	}
+	ShaderNetworkPtr networkCopy = shaderNetwork->copy();
+	IECoreScene::ShaderNetworkAlgo::convertToOSLConventions( networkCopy.get(), 10900 );
```

Comment:
As discussed, we would have to test not just whether the OSL version used by Appleseed supports component connections, but also whether Appleseed itself supports component connections.  Since our Appleseed support is on a deprecation path anyway, I won't worry about it here ... but it does highlight that driving this functionality based solely on an OSL version input is not totally ideal.